### DISCUSSION TITLE
Will powershell consider joining the .net foundation?

### DISCUSSION THREAD

**Discussion by kasini3000** [PowerShell/PowerShell#16007] [General]
Recently, I saw the news that the ownership of asp.net core has changed from microsoft to .net foundation.
It seems that this will make the project better, and I think powershellv7.2 can do the same.

Look from powerbi. In the past year, Linux powershell users seem to be decreasing.

Can powershell v7.2 change the ownership? And use this as a feature for publicity, what do you think?

List of projects in the .net foundation: (excluding powershell core)
https://dotnetfoundation.org/projects

**Comment by God-damnit-all** [comment]
Indeed, there are a lot of Linux diehards who won't touch PowerShell simply because Microsoft owns it. A switch to the .NET Foundation would likely generate some buzz on Linux-favoring tech news websites and get people talking about how useful it is.
